pkg/instancetype/instancetype.go

```diff
@@ -1221,13 +1222,17 @@ func applyDiskPreferences(preferenceSpec *instancetypev1beta1.VirtualMachinePref
 	}
 }
 
+func isInterfaceBindingUnset(interfaceBindingMethod virtv1.InterfaceBindingMethod) bool {
+	return reflect.ValueOf(&interfaceBindingMethod).Elem().IsZero()
+}
+
 func applyInterfacePreferences(preferenceSpec *instancetypev1beta1.VirtualMachinePreferenceSpec, vmiSpec *virtv1.VirtualMachineInstanceSpec) {
 	for ifaceIndex := range vmiSpec.Domain.Devices.Interfaces {
```

Comment:
Updated, appreciate this is documented in the user-guide but it might be useful if someone could also list these requirements out in the API docs?

https://kubevirt.io/api-reference/main/definitions.html#_v1_interface